gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -72,5 +94,42 @@ def start_postgres
     def register_standby_server
       run_repmgr_command(REGISTER_CMD)
     end
+
+    def configure_repmgrd
+      return true unless run_repmgrd_configuration
+      write_repmgrd_config
+      write_pgpass_file
+      start_repmgrd
+    end
+
+    def write_repmgrd_config
+      File.open(REPMGR_CONFIG, "a") do |f|
+        f.write("failover=automatic\n")
+        f.write("promote_command='repmgr standby promote'\n")
+        f.write("follow_command='repmgr standby follow'\n")
+        f.write("logfile=#{REPMGRD_LOG}\n")
+      end
```

Comment:
I found doing that a bit odd because even if I make this method return true, it still wont return false in the failure case. So when this method fails it wouldn't short circuit the suggested logic with `&&`, it would just raise anyway. And because this method is so unlikely to fail I feel like that's okay. And if it fails without raising, it is almost certain that `start_repmgrd` would fail, which has a well defined and handled error case.

I'm thinking I'll leave this and `configure_repmgrd` as is and just remove the pointless `true` from `write_pgpass_file`. Does that make sense?